src/wallet/run-wallet-example.js

```diff
@@ -1,40 +1,39 @@
 const WALLET_SIGNED_CREDENTIAL = {
```

Comment:
should example code be in a separate folder? Maybe a {package}/examples folder structure or something?

Reply:
I've removed this file, its a example for the universal wallet package. Its not required on this repo